.buildkite/macos/macos.rayci.yml

```diff
@@ -0,0 +1,130 @@
+group: macos tests
+sort_key: "~macos"
+steps:
+  # block on premerge
+  - block: "run macos tests"
+    if: build.env("BUILDKITE_PIPELINE_ID") == "0189942e-0876-4b8f-80a4-617f988ec59b"
+
```

Comment:
we still want to allow people to unblock and run tests on premerge pipeline right?

I am thinking, we should modify rayci into a way that we can select a sub graph of the whole graph (based on tags). I already have most of the work done (on deps tracking), but just need to add the feature we needed, and then we can simply have postmerge "block" macos tests (or just condition it out), and have the postmerge-macos only picking the "macos" tags.

or.. we can also just have a premerge-macos and a postmerge-macos .. which can be simpler on high-level structure, but less flexible down the road.

Reply:
nothing changes on the user side yes, `0189942e-0876-4b8f-80a4-617f988ec59b` is the premerge pipeline; this macos.rayci.yml is shared on both premerge and postmerge-macos pipeline

Reply:
ah, you have a symlink. ok.